src/js/media-container.js

```diff
@@ -304,51 +304,66 @@ class MediaContainer extends window.HTMLElement {
     // This allows things like autoplay and programmatic playing to also initiate hiding controls (CJP)
     this.setAttribute('user-inactive', 'user-inactive');
 
-    const scheduleInactive = () => {
+    const setInactive = () => {
+      if (this.autohide < 0) return;
+      this.setAttribute('user-inactive', 'user-inactive');
+      const evt = new window.CustomEvent(
+        MediaStateChangeEvents.USER_INACTIVE,
+        { composed: true, bubbles: true, detail: true }
+      );
+      this.dispatchEvent(evt);
+    };
+
+    const setActive = () => {
       this.removeAttribute('user-inactive');
       const evt = new window.CustomEvent(
-        MediaStateChangeEvents.USER_INACTIVE, 
+        MediaStateChangeEvents.USER_INACTIVE,
         { composed: true, bubbles: true, detail: false }
       );
       this.dispatchEvent(evt);
+    }
+
+    const autohide = () => {
```

Comment:
nit (blocking): Rename. Having a function called `autohide` that refers to a numeric property called `autohide` will be confusing.

Reply:
Going to move autohide back into scheduleInactive as it's no longer needed as a separate function.